doc/development/html_themes/index.rst

```diff
@@ -221,6 +221,60 @@ If your theme package contains two or more themes, please call
    ``sphinx.html_themes`` entry_points feature.
 
 
+Styling with CSS
+----------------
+The **stylesheets** setting allows to add custom CSS files to a theme.
```

Comment:
```suggestion
.. tip::

  The :confval:`stylesheets` setting can be used to add custom CSS files to a theme.
```

Reply:
I've chosen to use the text but not put it in an admonition. First, the sentence describes the foundation of styling; I feel tips should be useful additional but not essential pieces of information. Second, I've turned the following paragraph into a `.. caution`. Having a section only consisting of two admonitions feels weird. There should be some "normal" information.